src/arc.ts

```diff
@@ -124,9 +125,35 @@ function getBiconomy(provider: any): any {
   return biconomy;
 }
 
+
+/**
+ * Function to show user contract wallet address to inform him that he/she
+ * should make sure to have reputation in this address in order to vote for dao
+ * without paying transaction fee.
+ * @param userContract User contract wallet address
+ */
+function showUserContractAddress(userContract: string, options?: IEnableWalletProviderParams) {
+  if (userContract) {
+    // eslint-disable-next-line no-console
+    console.info(`User contract wallet ${userContract}`);
+    if (options)
```

Comment:
This options should be removed as its not being used now.
Because of this when user changes the account in metamask, and again signing the user login message, this popup message is not showing.

Reply:
I modified the function arguments to handle the two cases which were with and without options.